src/app/modules/core/components/map/map.component.ts

```diff
@@ -36,12 +36,31 @@ export class MapComponent extends deepCopy implements OnInit, AfterViewInit {
   public layerGroup;
   public reportlayerGroup;
   public map: L.Map;
+  public isfirst = true;
+
 
   public evnt;
   @Input() report: boolean;
+  @Input() mapSize: string;
 
   scaleMap: string;
 
+
+  ngOnChanges(changes: any) {
+	// If map size changed, trigger resize event, which will force map to redraw
```

Comment:
Is a resize event triggered when the footer section pops up? I am unable to see all flowline segments shown in the map when the footer pops up.